torch_xla/csrc/runtime/stablehlo_composite_helper.cc

```diff
@@ -350,9 +397,13 @@ class BuildStableHLOCompositePass : public mlir::OperationPass<mlir::ModuleOp> {
       mlir::Operation* cloned_op = builder.clone(*original_op, mapping);
```

Comment:
QQ: Is this where we make sure the arg in the composite impl func is connecting to the correct impl ops? The cloned op can find the correct input op by looking up the `IRMapping`, in which the mapping from the ops in the original graph to the new impl func is stored?

Reply:
true. `build.clone` replace `original_op` operands with the new cloned operands by looking up records in `mapping`.